airbyte-integrations/connectors/source-harness/metadata.yaml

```diff
@@ -25,67 +25,66 @@ data:
   supportLevel: community
   documentationUrl: https://docs.airbyte.com/integrations/sources/harness
   tags:
-  - cdk:low-code
-  - language:manifest-only
+    - cdk:low-code
+    - language:manifest-only
   connectorTestSuitesOptions:
-  - suite: acceptanceTests
```

Comment:
Hmmmm, we should keep the test configurations in test suites. These determine which tests CI would run.